packages/fxa-content-server/tests/functional/fx_ios_v1_sign_in.js

```diff
@@ -53,7 +53,7 @@ const setupTest = thenify(function(options = {}) {
   const successSelector = options.blocked
     ? selectors.SIGNIN_UNBLOCK.HEADER
     : options.preVerified
-    ? selectors.CONFIRM_SIGNIN.HEADER
```

Comment:
I wonder how much value `signin token code` language has now since we have the `confirm signin` views available.